docs/get-started/installation/prepare/microcontrollers/development-setup.md

```diff
@@ -242,12 +242,14 @@ Follow [this guide](/build/micro-rdk/board/esp32/) to configure an `esp32` board
 [Start Docker](https://docs.docker.com/config/daemon/start/) on your development machine.
 If you haven't already, complete Docker's [Linux post installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to set up Docker to run whenever your system boots up.
 
-Use [the Micro-RDK template](https://github.com/viamrobotics/micro-rdk-robot-template.git) to create a new Micro-RDK project to upload to your ESP32 by running:
+Use [the Micro-RDK template](https://github.com/viamrobotics/micro-rdk/tree/main/templates/project) to create a new Micro-RDK project to upload to your ESP32 by running:
 
 ```sh { class="command-line" data-prompt="$"}
-cargo generate --git https://github.com/viamrobotics/micro-rdk-robot-template.git
+cargo generate --git https://github.com/viamrobotics/micro-rdk.git
 ```
 
+And select `templates/project` when prompted.
```

Comment:
There is a new step now where cargo generate will ask to generate a module or a project. The user needs to select project afterwards the steps are the same as before.

If this is not the best way the represent the new flow happy to change it. If the explanation is unclear let me know!

Reply:
This sounds good, just would change the wording a tiny bit!

```suggestion
Select `templates/project` when prompted.
```